diff --git a/lib/daemons/spell_d.c b/lib/daemons/spell_d.c
index 53a4372..09e29b3 100644
--- a/lib/daemons/spell_d.c
+++ b/lib/daemons/spell_d.c
@@ -208,8 +208,8 @@ void register_spell()
    string spell_name = previous_object()->qu_spell_name();
    string spell_info = spell_table[spell_name];
 
-   ENSURE(spell_name /* You didn't provide a spell name. */);
-   ENSURE(valid_spell_name(spell_name) /* Your spell name wasn't valid. */);
+   ENSURE(spell_name); /* You didn't provide a spell name. */
+   ENSURE(valid_spell_name(spell_name)); /* Your spell name wasn't valid. */
 
    if (!spell_info)
    {
