diff --git a/lib/std/race/troll.c b/lib/std/race/troll.c
index fc2e6de..23d54ec 100644
--- a/lib/std/race/troll.c
+++ b/lib/std/race/troll.c
@@ -22,7 +22,8 @@ string *compatibility()
  */
 string short_description()
 {
-   return @LONG Trolls are extremely strong and powerful creatures; probably the toughest
+   return @LONG
+Trolls are extremely strong and powerful creatures; probably the toughest
 race available. Their skin is extremely thick and conventional means of
 damage do not help much in killing them. Trolls aren't very intelligent
 or quick on their feet, but have an amazingly high willpower and just
