diff --git a/lib/domains/std/room/Attic.c b/lib/domains/std/room/Attic.c
index 0a2427c..98e5ee8 100644
--- a/lib/domains/std/room/Attic.c
+++ b/lib/domains/std/room/Attic.c
@@ -1,4 +1,3 @@
-#pragma no_warnings
 inherit "/std/indoor_room";
 
 void setup();
diff --git a/lib/domains/std/room/Church.c b/lib/domains/std/room/Church.c
index d049981..3a0201e 100644
--- a/lib/domains/std/room/Church.c
+++ b/lib/domains/std/room/Church.c
@@ -1,4 +1,3 @@
-#pragma no_warnings
 inherit "/std/indoor_room";
 
 void setup();
diff --git a/lib/domains/std/room/wiz_hall.c b/lib/domains/std/room/wiz_hall.c
index b1fbb7c..fffdedb 100644
--- a/lib/domains/std/room/wiz_hall.c
+++ b/lib/domains/std/room/wiz_hall.c
@@ -1,4 +1,3 @@
-#pragma no_warnings
 inherit "/std/indoor_room";
 
 void setup();
diff --git a/lib/include/mudlib.h b/lib/include/mudlib.h
index 89e1a43..1e61bc2 100644
--- a/lib/include/mudlib.h
+++ b/lib/include/mudlib.h
@@ -2,7 +2,6 @@
 
 #ifndef __MUDLIB_H__
 #define __MUDLIB_H__
-#pragma no_warnings
 
 #include <daemons.h>
 #include <dirs.h>
